css/swc.css

```diff
@@ -18,11 +18,6 @@ div.chapter h2 {
     font-style: italic;
 }
 
-/* Objectives and key points */
-.objectives, .keypoints {
```

Comment:
Why did you remove `.objectives`? We drop `.keypoints` but `.objectives` wasn't drop (see https://raw.githubusercontent.com/swcarpentry/lesson-template/gh-pages/01-one.md).